zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java

```diff
@@ -34,6 +40,8 @@
  * Override the {@link org.jboss.seam.security.FacesSecurityEvents} component to
  * alter default values.
  *
+ * TODO [CDI] need to find a CDI replacement.
```

Comment:
Okay.

We don't need to do it for this PR, but how are we going to replace StatusMessage(s)?  From the class name, it sounds like it should overlap with FacesMessages, but it doesn't seem to.  I can't work out what it actually does with the messages after interpolating the variables.  Wait: what it does is add them to the global `messages` list, Seam's FacesMessages extends StatusMessages, and it displays any waiting global messages in `beforeRenderResponse()`.  

So FacesSecurityEvents could generate the status messages and call our `FacesMessages.addGlobal` to achieve the same effect.  

- [ ] @huangp If you agree, could you please put in a TODO to that effect?